requirements-dev.txt

```diff
@@ -1,77 +1,83 @@
 -i https://pypi.org/simple
-alabaster==0.7.13; python_version >= '3.6'
-attrs==22.2.0; python_version >= '3.6'
-babel==2.11.0; python_version >= '3.6'
+alabaster==0.7.13 ; python_version >= '3.6'
```

Comment:
Were the changes in this file required by `bandit`?

Reply:
yes, it was added by the bandit.